docs/source/developer/domain-server/environment-variables.md

```diff
@@ -0,0 +1,39 @@
+# Environment Variables
+
+You can alter the domain server's functionality using the following environment variables.
```

Comment:
```suggestion
You can alter the Domain server's functionality using the following environment variables.
```
My understanding is that we use "Domain" as a name, rather than a normal noun that is found in a dictionary.

Reply:
Looking at the docs, it seems High Fidelity used it as "domain" rather than "Domain" the whole way through.

Reply:
My understanding is that the way High Fidelity did this is grammatically wrong. Not sure if High Fidelity doing something wrong would warrant us doing the same. Wouldn't be the first thing they did wrong in the docs. We also have to go through pretty much all doc pages anyways, so fixing this seems trivial.

Reply:
I agree, was just taking note of what they did. If we capitalize "Interface" then I suppose "Domain" should follow a similar convention if the word is being used in a similar way.